src/python/pants/backend/scala/target_types.py

```diff
@@ -51,13 +51,17 @@ class ScalaDependenciesField(Dependencies):
 
 
 class ScalaConsumedPluginNamesField(StringSequenceField):
-    """The names of Scala plugins that this source file requires.
+    help = """The names of Scala plugins that this source file requires.
 
-    If not specified, this will default to the plugins specified in `--scalac-plugins` for this
-    target's resolve.
-    """
+        The plugin must be defined by a corresponding `scalac_plugin` AND `jvm_artifact` target,
+        and must be present in this target's resolve's lockfile.
+
+        If not specified, this will default to the plugins specified in `--scalac-plugins` for this
+        target's resolve.
+        """
```

Comment:
This should be using implicit string concatenation. That's important so that Pants can resize the output based on your terminal width via soft wrapping. Whereas this uses hard wrapping set by the plugin author.

I'll fix in followup rn.